generateDB.py

```diff
@@ -12,15 +12,17 @@ def generate_db_from_json(manga_json, sites_json, db_name='mycheckmanga.db'):
         cur.execute("DROP TABLE IF EXISTS manga")
         cur.execute("DROP TABLE IF EXISTS sites")
         cur.execute("DROP TABLE IF EXISTS tags")
+        cur.execute("DROP TABLE IF EXISTS completed")
         cur.execute("CREATE TABLE sites(SiteName TEXT PRIMARY KEY)")
         cur.execute("CREATE TABLE tags(ID INT PRIMARY KEY, Site TEXT, TagName TEXT, TagClass TEXT, FOREIGN KEY(Site) references sites(SiteName))")
-        cur.execute("CREATE TABLE manga(Title TEXT PRIMARY KEY, Site TEXT, Status TEXT, Url TEXT, LastChapterRead TEXT, MostRecentChapter TEXT, FOREIGN KEY(Site) references sites(SiteName))")
+        cur.execute("CREATE TABLE manga(Title TEXT PRIMARY KEY, Site TEXT, Status TEXT, Url TEXT, LastChapterRead TEXT, TimeLastUpdated TEXT, TimeLastChecked TEXT, MostRecentChapter TEXT, CoverImageUrl TEXT, FOREIGN KEY(Site) references sites(SiteName))")
```

Comment:
file like this could be good to require a password or a confirmation or something, bc it can easily overwrite your entire db